spearhead/variables.css

```diff
@@ -108,10 +109,11 @@
 		--global--color-foreground-low-contrast: #b2b2b2;
 		--global--color-background: #1e1f21;
 		--global--color-text-selection: #000000;
-		--primary-nav--color-background: var(--global--color-foreground-low-contrast);
-		--button--padding-vertical: 24px;
-		--button--padding-horizontal: 32px;
 		--sticky-posts--color-background: var(--global--color-background);
 		--sticky-posts--alt-color-background: #2d3139;
+		--primary-nav--color-background: var(--sticky-posts--alt-color-background);
+		--primary-nav--dropdown-color-link: var(--global--color-foreground);
+		--button--padding-vertical: 24px;
+		--button--padding-horizontal: 32px;
```

Comment:
Why are you only changing the button padding in dark mode?

Reply:
No, I'm  not changing the padding, I changed the variable order only. The padding values are the same.

Reply:
Oh yeah I see now